src/web/components/List/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Item from '../Item';
+
+export default function List({ data, renderItem, keyName }) {
+  if (data.length < 1) {
```

Comment:
you can also do 

```javascript
if (!data.length) {
}
```

Reply:
Thanks I will stop using my noob method :smile: